### PR TITLE
Add OperatorSpacingSniff

### PR DESCRIPTION
**Pull request by grongor**
Part of #701 series.

:wrench: TODO: add readme if sniff is accepted

### CODE REVIEW

File: SlevomatCodingStandard/Sniffs/Operators/OperatorSpacingSniff.php [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+use const T_PLUS;
+use const T_PRINT;
+use const T_STRING_CAST;
+use const T_UNSET_CAST;
+use const T_YIELD;
+
+/**
+ * We need this sniff until Squiz accepts fix for unary operands detection
+ * https://github.com/squizlabs/PHP_CodeSniffer/pull/2640
+ */
```
Comment (kukulich) [inline SlevomatCodingStandard/Sniffs/Operators/OperatorSpacingSniff.php]: Is this comment still valid? PR is merged.

File: SlevomatCodingStandard/Sniffs/Operators/OperatorSpacingSniff.php [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+final class OperatorSpacingSniff extends SquizOperatorSpacingSniff
+{
+
+	private const NON_OPERAND_TOKENS = [
+		T_ECHO,
+		T_PRINT,
+		T_YIELD,
+
+		T_ARRAY_CAST,
```
Comment (kukulich) [inline SlevomatCodingStandard/Sniffs/Operators/OperatorSpacingSniff.php]: Please remove space.

File: SlevomatCodingStandard/Sniffs/Operators/OperatorSpacingSniff.php [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+use const T_PRINT;
+use const T_STRING_CAST;
+use const T_UNSET_CAST;
+use const T_YIELD;
+
+/**
+ * We need this sniff until Squiz accepts fix for unary operands detection
+ * https://github.com/squizlabs/PHP_CodeSniffer/pull/2640
+ */
+final class OperatorSpacingSniff extends SquizOperatorSpacingSniff
```
Comment (kukulich) [inline SlevomatCodingStandard/Sniffs/Operators/OperatorSpacingSniff.php]: No sniff is final in this repo.

File: SlevomatCodingStandard/Sniffs/Operators/OperatorSpacingSniff.php (hunk 1 above)
Comment (grongor) [reply SlevomatCodingStandard/Sniffs/Operators/OperatorSpacingSniff.php]: You are correct, sorry for not checking that. This sniff is no longer needed.

**Pull request closed without merging**